### PR TITLE
Update ruff lint rules

### PR DESCRIPTION
**Pull request by rocket3989**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,7 +62,7 @@ cover/
 .ruff_cache/
 
 # Django stuff:
-*.log
+*.log*
```
Comment (kzfubar) [inline .gitignore:65]: what additional file types is this used for?

File: server.py [hunk 2]
```diff
@@ -99,46 +108,46 @@ def add_user_info_to_config(user_id, user_email, access_token):
...
         task = api.get_task(completed_task_id)
-        if task is None:
-            print(f"Task with ID {completed_task_id} not found.")
+        if task is None or task.labels is None:
             return
 
         updated_labels = [label for label in task.labels if label != "shame"]
         success = api.update_task(task_id=completed_task_id, labels=updated_labels)
 
         if success:
```
Comment (kzfubar) [inline server.py:145]: if success
pass
else
pass

Comment (kzfubar) [reply server.py:145]: maybe we should log here

File: shame_script.py [hunk 3]
```diff
@@ -30,7 +31,7 @@
 
 TODOIST_API = "https://api.todoist.com/rest/v2/tasks"
 
-SCHEDULED_UTC_POST_TIME = time(hour=2)
+SCHEDULED_UTC_POST_TIME = time(hour=0, minute=27)
```
Comment (kzfubar) [inline shame_script.py]: did we want to change this?

**Review by kzfubar — COMMENTED** [review]
left some comments

File: .gitignore (hunk 1 above)
Comment (rocket3989) [reply .gitignore:65]: the `TimedRotatingFileHandler` adds `.date` to the end of backups

File: shame_script.py (hunk 3 above)
Comment (rocket3989) [reply shame_script.py]: oops

File: .gitignore (hunk 1 above)
Comment (kzfubar) [reply .gitignore:65]: ah nice

**Review by kzfubar — APPROVED** [review]
+2

**Pull request merged**